lib/metalsmith-lunr-index/index.js

```diff
@@ -63,6 +93,16 @@ module.exports = function lunrPlugin () {
         }
         this.add(doc)
       })
+      // store all headings into the Lunr store
+      pageHeadingDocuments.forEach(doc => {
```

Comment:
At the moment if you type 's', Screen size comes up first, should we weight these lower than page titles?

cc @dashouse

Reply:
ATM they're the same type of docs are pages. to do weight, they will have to be have different fields to be weighted againts

Reply:
That sounds tricky, but I think it could cause issues having something lower in the hierarchy come up first?

<img width="318" alt="screen shot 2018-09-17 at 10 55 56" src="https://user-images.githubusercontent.com/2445413/45616756-4910c100-ba68-11e8-8a57-dd00b0768538.png">

Maybe not a big deal...

Reply:
I think from the users perspective the things they're going to see here are conceptually on the same level even if they are technically lower in the hierarchy. For example "Lists" and "Inset text". The "screen size" one doesn't feel like it I agree but generally I think it's the right approach. We may might need to re-think some of the h2's now though...